### DISCUSSION TITLE
Add here the repo and website for Lesson 4

### DISCUSSION THREAD

**Discussion by igordsm** [Insper/open-dev#775] [Show and tell]
Each group must add a post here with their project repository and website before sending a PR to complete the skill

**Comment by jpgianfaldoni** [comment]
Repository: https://github.com/jpgianfaldoni/CookieJar/tree/main
Documentation: https://jpgianfaldoni.github.io/CookieJar/

**Reply by FidyBack** [reply]
Escolhi esse

**Comment by FidyBack** [comment]
Repository: https://github.com/FidyBack/Projeto_Z_de_Jogo
Documentation: https://fidyback.github.io/Projeto_Z_de_Jogo

**Reply by jpgianfaldoni** [reply]
Escolhi esse

**Reply by cemmanuelsr** [reply]
Vou escolher esse pra atividade "Automated testing"

**Reply by hsalberti** [reply]
Vou fazer esse para  "Code Quality"

**Comment by hsalberti** [comment]
Repository: https://github.com/nataliaqmc/Projeto_Final_DesSoft
Documentation: https://nataliaqmc.github.io/Projeto_Final_DesSoft/

**Reply by FidyBack** [reply]
Vou escolher esse pra atividade "Automated testing"

**Comment by JorasOliveira** [comment]
repository: https://github.com/JorasOliveira/pygame_refactor
Documentation: https://jorasoliveira.github.io/pygame_refactor/

**Reply by jpgianfaldoni** [reply]
Vou fazer o automated testing desse

**Reply by cemmanuelsr** [reply]
@igordsm vou mudar minha escolha de "Code quality" pra esse porque não achei issue aberta no outro

**Comment by Antonioarf** [comment]
Rep: https://github.com/Antonioarf/trabalho_desoft
Docs: https://antonioarf.github.io/trabalho_desoft

**Reply by JorasOliveira** [reply]
escolhi esse para a atividade de 'Code Quality'

**Comment by cicerotcv** [comment]
Rep: [github.com/cicerotcv/megadados/](https://github.com/cicerotcv/megadados/)
Docs: [https://cicerotcv.github.io/megadados/](https://cicerotcv.github.io/megadados/)
